scipy/ndimage/_filters.py

```diff
@@ -1490,6 +1499,24 @@ def convolve(input, weights, output=None, mode='reflect', cval=0.0,
            [5, 0, 2],
            [3, 0, 1]])
 
+    The `origin` parameter controls filter placement. Here's a simple example:
+
+    >>> d = np.array([[1, 2, 3],
+    ...               [4, 5, 6], 
+    ...               [7, 8, 9]])
+    >>> k_simple = np.array([[0, 1], 
```

Comment:
`k_simple` seems to me to be a confusing name, because of the use of $k$ in the explanation of filter centering.

Maybe `w_simple` (w for weights).